Sources/eppo/RuleEvaluator.swift

```diff
@@ -3,131 +3,259 @@ import Semver
 
 typealias ConditionFunc = (Double, Double) -> Bool;
 
-class Compare {
-    public static func compareNumber(_ a: Double, _ b: Double, _ conditionFunc: ConditionFunc) -> Bool {
-        return conditionFunc(a, b);
-    }
-
+class Compare {    
     public static func compareRegex(_ a: String, _ pattern: String) -> Bool {
         return a.range(of: pattern, options:.regularExpression) != nil;
     }
-
+    
     public static func isOneOf(_ a: String, _ values: [String]) -> Bool {
         return values.map({ $0.lowercased() })
-                     .contains(a.lowercased());
+            .contains(a.lowercased());
```

Comment:
UFC is now fully case-sensitive so we should no longer lowercase.

Reply:
Adding a test for this in the sdk-test-data repo

Reply:
I see this in python too: https://github.com/Eppo-exp/python-sdk/blob/main/eppo_client/rules.py#L59-L61